links.html

```diff
@@ -26,9 +26,9 @@
         <h1>Useful Links</h1>
         <ul>
             <li><a href="https://validator.w3.org/nu/" target="_blank">HTML Validator</a></li>
-            <li><a href="https://jigsaw.w3.org/css-validator/" target="blank">CSS Validator</a></li>
-            <li><a href="https://developer.mozilla.org/en-US/" target="blank">MDN</a></li>
-            <li><a href="https://caniuse.com/" target="blank">Can I Use</a></li>
+            <li><a href="https://jigsaw.w3.org/css-validator/" target="_blank">CSS Validator</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/" target="_blank">MDN</a></li>
+            <li><a href="https://caniuse.com/" target="_blank">Can I Use</a></li>
```

Comment:
"blank" vs "_blank" is important: https://stackoverflow.com/questions/35703005/what-is-the-difference-between-target-blank-and-target-blank